File: nan21.dnet.module.ad/nan21.dnet.module.ad.ui.extjs/src/main/webapp/src/ds/ImportMapLovDs.js
 
   	 
Ext.define("net.nan21.dnet.module.ad.impex.ds.model.ImportMapLovDs" , {
	extend: 'Ext.data.Model',
	fields: [  		
		{name:"id",type:"int", useNull:true }, 		
		{name:"clientId",type:"int", useNull:true }, 		
		{name:"name",type:"string" }, 		
		{name:"active",type:"boolean" }
	]
});
	
Ext.define("net.nan21.dnet.module.ad.impex.ds.param.ImportMapLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 

File: nan21.dnet.module.ad/nan21.dnet.module.ad.presenter/src/main/java/net/nan21/dnet/module/ad/impex/ds/model/ImportMapLovDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms. 
 */
package net.nan21.dnet.module.ad.impex.ds.model;

import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeLov;
import net.nan21.dnet.module.ad.impex.domain.entity.ImportMap;

@Ds(entity = ImportMap.class, jpqlWhere = " e.active = true ", sort = { @SortField(field = ImportMapLovDs.fNAME) })
public class ImportMapLovDs extends AbstractTypeLov<ImportMap> {

    public ImportMapLovDs() {
        super();
    }

    public ImportMapLovDs(ImportMap e) {
        super(e);
    }

}
